src/plugins/condition/components/Criterion.vue

```diff
@@ -162,6 +161,19 @@ export default {
                 }
             }
             return type;
+        },
+        inputCount: function () {
+            let inputCount = 0;
+            if (this.criterion.operation) {
+                for (let i = 0; i < this.filteredOps.length; i++) {
+                    if (this.criterion.operation === this.filteredOps[i].name) {
+                        inputCount = this.filteredOps[i].inputCount;
+                        break;
+                    }
+                }
+            }
+            this.checkTelemetry();
```

Comment:
A `computed` should never modify any reactive data. `this.checkTelemetry()` will do just that. Why would we need to checkTelemetry when getting inputCount anyways? it should just be providing you with the current state, not modifying it.

Reply:
Took this out. With the switch of the index in v-for in Condition to render Critieria it's not needed.